scss/omni-nav/omni-nav.scss

```diff
@@ -0,0 +1,397 @@
+@import "variables";
+@import "mixins";
+@import "icomoon";
+
+html {
+  padding-top: $primary-nav-height;
+
+  // Global and primary nav are stacked only on tablet
+  @include breakpoint($bp-tablet) {
+    padding-top: 120px;
+  }
+
+  @include breakpoint($bp-large-desktop) {
```

Comment:
Notice the change in breakpoint here (switched from 1300px to 1190px). See [trello comment](https://trello.com/c/qIH4kmWs/390-as-a-user-of-chapmans-websites-id-like-to-see-the-omninav-updated-to-match-the-new-design-on-www-omninav-as-a-service#comment-5a999e6086719bee7d4e7af0).